### PR TITLE
replace mitre references with minimart

### PR DESCRIPTION
**Pull request by jblefkove-va**
- Rename `local-mitre-db.sh` to `export-h2.sh`
- Rename `DatamartExporter` to `H2Exporter`
- Update assorted references from `MITRE` to `Minimart`
- Rename `MitreMinimartMaker` to `MinimartMaker`
- Add `minimart-manager/README.md` and move `Augment` documentation
- Rename `Minifier` to `IdValidationAugment`

### CODE REVIEW

File: minimart-manager/export-h2.sh [hunk 1]
```diff
@@ -1,11 +1,10 @@
 #!/usr/bin/env bash
```
Comment (taylorrassmannlibertyits) [inline minimart-manager/export-h2.sh:1]: Were the file permissions supposed to change here?

Comment (jblefkove-va) [reply minimart-manager/export-h2.sh:1]: no that's probably windows

File: minimart-manager/src/test/java/gov/va/api/health/minimartmanager/H2Exporter.java [hunk 2]
```diff
@@ -70,33 +66,30 @@ public class DatamartExporter {
           ExportForPatientCriteria.of(ProcedureEntity.class),
           ExportAllCriteria.of(VitalVuidMappingEntity.class));
 
-  EntityManager h2;
+  EntityManager minimart;
 
-  EntityManager mitre;
+  EntityManager h2;
```
Comment (taylorrassmannlibertyits) [inline minimart-manager/src/test/java/gov/va/api/health/minimartmanager/H2Exporter.java:71]: lunar alphabetical order?

Comment (jblefkove-va) [reply minimart-manager/src/test/java/gov/va/api/health/minimartmanager/H2Exporter.java:71]: Source then destination

**Review by taylorrassmannlibertyits — DISMISSED** [review]

**Review by taylorrassmannlibertyits — APPROVED** [review]

**Pull request merged**